### PR TITLE
super-productivity: init at 1.10.45

### PR DESCRIPTION
**Pull request by offlinehacker**

### CODE REVIEW

File: pkgs/applications/networking/super-productivity/default.nix [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  buildInputs = [
+    dpkg
+    gtk3  # needed for GSETTINGS_SCHEMAS_PATH
+  ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  unpackPhase = "true";
+  buildCommand = ''
```
Comment (adisbladis) [inline pkgs/applications/networking/super-productivity/default.nix]: It's a bit awkward to have all phases in one like this, makes overriding difficult.

File: pkgs/applications/networking/super-productivity/default.nix [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    makeWrapper $out/libexec/superProductivity/superproductivity $out/bin/superproductivity \
+      --prefix XDG_DATA_DIRS : $GSETTINGS_SCHEMAS_PATH \
+      --prefix PATH : "${xprintidle-ng}/bin"
+
+    # Fix the desktop link
+    substituteInPlace $out/share/applications/superproductivity.desktop \
+      --replace /opt/superProductivity/ $out/bin/
+  '';
+
+  meta = with stdenv.lib; {
```
Comment (adisbladis) [inline pkgs/applications/networking/super-productivity/default.nix]: No maintainer?

File: pkgs/applications/networking/super-productivity/default.nix (hunk 1 above)
Comment (offlinehacker) [reply pkgs/applications/networking/super-productivity/default.nix]: I just copied slack derivation, i can put it in install phase nad disable other phases, do you think that would be better?

File: pkgs/applications/networking/super-productivity/default.nix (hunk 2 above)
Comment (offlinehacker) [reply pkgs/applications/networking/super-productivity/default.nix]: Well, again, slack package did not have maintainer, so i forgot.

**Comment by offlinehacker** [comment]
Ok, i split into `unpackPhase` and `installPhase`, i also called `preInstall` and `postInstall` hooks from `installPhase`. For `fixupPhase` i had to disable striping. Also added me as maintainer.

**Comment by adisbladis** [comment]
Much better. Thanks :)

**Pull request merged**